yogstation/code/modules/clothing/head/jobs.dm

```diff
@@ -8,14 +8,12 @@
 //Mining Medic
 /obj/item/clothing/head/soft/emt/mining
 	name = "Mining Medic's cap"
-	desc = "It's a baseball hat with a dark turquoise color and a reflective cross on the top."
-	icon = 'yogstation/icons/obj/clothing/hats.dmi'
-	alternate_worn_icon = 'yogstation/icons/mob/head.dmi'
+	desc = "It's a baseball hat with a dark turquoise color and a reflective cross on the top. Has MM embrossed into it."
 
 //Brig Physician
 /obj/item/clothing/head/soft/emt/phys
 	name = "Brig Physician's cap"
-	desc = "It's a baseball hat with a dark brown color and a reflective cross on the top. On the back are "
+	desc = "It's a baseball hat with a dark brown color and a reflective cross on the top. Has BP embrossed into it."
```

Comment:
```suggestion
	desc = "It's a baseball hat with a dark brown color and a reflective cross on the top. Has BP embossed into it."
```

Reply:
Well, I copied that from the mining medic jacket and expected it to not have typos, my bad. I guess Ill go fix those too while im at it.